.isort.cfg

```diff
@@ -0,0 +1,2 @@
+[settings]
```

Comment:
I have no idea what this file is but I'm guessing it's ok to be checked in?

Reply:
Yeah, it popped up in the api setup, IDK what it's for but it's there.